### PR TITLE
Fix up storage plus CLI

### PR DESCRIPTION
**Pull request by pmatulis**

### CODE REVIEW

File: en/installconfig-storage.md [hunk 1]
```diff
@@ -1,79 +1,75 @@
...
-| sda1      | 512M     | part  | fat32         | /boot/efi    |
-| sda2      | 99.5G    | part  | lvm-pv(vgroot)|              |
-| vgroot    | 99.5G    | lvm   |               |              |
-| lvroot    | 99.5G    | lvm   | ext4          |  /           |
+## UEFI
 
-The following options are supported for this layout. :
+A node booting with UEFI will be accommodated by every layout type by having it
+include an EFI boot partition (`/boot/efi`) in its configuration. Other than
+setting the node to boot from UEFI, no other action is asked of the user.
```
Comment (degville) [inline en/installconfig-storage.md]: This makes better sense the other way around to me. ie, something more like:
Including an EFI boot partition (`/boot/efi`) in a node's configuration will allow UEFI booting to be adopted by every layout type.

File: en/installconfig-storage.md [hunk 2]
```diff
@@ -1,79 +1,75 @@
...
 
-MAAS has the ability to configure any storage layout during node deployment.
-MAAS doesn't just do simple partitioning - it supports complex storage layouts,
-including setting up and configuring Bcache, RAID, and LVM. It also offers
-fine-grained control over the creation, deletion, formatting and mounting of
-both block devices and partitions (see [Block devices][block-devices] and
-[Partitions][partitions]).
+The final storage configuration that a deployed node uses can be influenced
+significantly. MAAS supports traditional disk partitioning as well as more
+complex options such as LVM, RAID, and Bcache. UEFI is also supported.
```
Comment (degville) [inline en/installconfig-storage.md]: The addition of UEFI seems a little redundant here. Maybe remove it, or say UEFI booting alongside BIOS booting?

File: en/installconfig-storage.md [hunk 3]
```diff
@@ -82,25 +78,56 @@ boot disk.
...
+the volume group.
+
+### Bcache layout
+
+A Bcache layout will create a partition that spans the entire boot disk as the
+backing device. It uses the smallest block device tagged with 'ssd' as the
+cache device. The Bcache device is formatted with the ext4 filesystem and uses
+the `/` mount point. If no block devices exists on the node that are tagged
+with 'ssd' then the Bcache device will not be created and the Flat layout will
+be used instead:
```
Comment (degville) [inline en/installconfig-storage.md]: If there are no 'ssd' tagged block devices on the node, then the Bcache device... ?

File: en/installconfig-storage.md [hunk 4]
```diff
@@ -126,58 +153,58 @@ Default is false, meaning to create a partition using the given `cache_size`.
...
+All nodes will have a default layout applied when commissioned. An
+administrator can configure the default layout on the 'Settings' page, under
+the 'General' tab. The section is labelled 'Storage':
 
-## Setting the Layout
+![default storage layout][img__2.2_default-storage-layout]
 
-It's also possible to change the storage layout either globally, on acquire, or
-after acquire.
+See [Disk erasure][storage-erasure] for explanation of the options related to
```
Comment (degville) [inline en/installconfig-storage.md]: "for explanation" is a little too terse for me. Maybe add an `an`?

File: en/installconfig-storage.md (hunk 2 above)
Comment (pmatulis) [inline en/installconfig-storage.md]: Why do you see it as redundant? Are you saying that a user expects UEFI support?

File: en/installconfig-storage.md (hunk 1 above)
Comment (pmatulis) [inline en/installconfig-storage.md]: No, it's the layout that will cause the EFI partition to be created.

File: en/installconfig-storage.md (hunk 3 above)
Comment (pmatulis) [inline en/installconfig-storage.md]: OK

File: en/installconfig-storage.md (hunk 4 above)
Comment (pmatulis) [inline en/installconfig-storage.md]: Right, typo.

File: en/installconfig-storage.md (hunk 1 above)
Comment (degville) [inline en/installconfig-storage.md]: I still find it the wrong way around. So, including EFI in the layout will mean that UEFI will be accommodated by every layout type?

File: en/installconfig-storage.md (hunk 2 above)
Comment (degville) [inline en/installconfig-storage.md]: It just doesn't seem to fall into the same category as LVM, RAID and Bcache, which made me think it needed either further clarification or dropping (as, yes, you'd expect UEFI support).

File: en/installconfig-storage.md (hunk 1 above)
Comment (pmatulis) [inline en/installconfig-storage.md]: No matter which layout is chosen, a EFI partition will be created if the node is booting with UEFI. I'll reword.

File: en/installconfig-storage.md (hunk 2 above)
Comment (pmatulis) [inline en/installconfig-storage.md]: I wanted to include UEFI somewhere in the introduction since there is a section about it. Suggestions?

Comment (degville) [inline en/installconfig-storage.md]: Maybe just `UEFI is also supported as a boot mechanism` to differentiate it from the storage operations?

Comment (pmatulis) [inline en/installconfig-storage.md]: OK

**Pull request merged**